tests/e2e/test_move_task.py

```diff
@@ -14,53 +14,63 @@ def test_move_task_to_column(browser: Page, db: DatabaseTestUtils) -> None:
     all_status = all_statusses()
     db.given([*all_status])
     db.given([default_task(status_id=all_status[0].id)])
+    db.given([default_user()])
 
-    browser.goto("/pages/")
+    page.goto("/pages/")
 
-    expect(browser.locator("#column-1 #card-container-1")).to_be_visible()
-    expect(browser.locator("#column-3")).to_be_visible()
+    card = page.get_by_role("heading", name="Default Task")
+    expect(card).to_be_visible()
+
+    expect(page.locator("#column-1")).to_be_visible()
+    expect(page.locator("#column-3")).to_be_visible()
+
+    with page.expect_response("/tasks/", timeout=1000) as response_info:
+        page.drag_and_drop("#card-container-1", "#column-3", target_position={"x": 50, "y": 50})
 
-    # todo (Robbert) action below is performed twice, because once does not work, we need find out why and fix it
-    browser.locator("#card-container-1").drag_to(browser.locator("#column-3"))
-    browser.locator("#card-container-1").drag_to(browser.locator("#column-3"))
+    response = response_info.value
+    assert response.status == 200
 
-    browser.reload()
+    page.reload()
 
-    card = browser.locator("#column-3 #card-container-1")
-    expect(card).to_have_id("card-container-1")
+    card = page.locator("#column-3 > #card-container-1")
     expect(card).to_be_visible()
 
 
-def test_move_task_order_in_same_column(browser: Page, db: DatabaseTestUtils) -> None:
+def test_e2e_move_task_order_in_same_column(page: Page, db: DatabaseTestUtils) -> None:
     """
-    Test moving a task in the browser below another task and verify that after a reload
+    Test moving a task in the page below another task and verify that after a reload
     it is in the right position in the column.
     :return: None
     """
     all_status = [*all_statusses()]
     db.given([*all_status])
 
-    task1 = default_task(title="Task 1", status_id=all_status[0].id, sort_order=10)
```

Comment:
Why did you remove the sort_order? This is needed to make sure the tasks are ordered correctly, the default_task gives them all a sort_order of 1. So I am surprised this test still works?!

Reply:
Ah. It probably works because you move card 1 below card 3. It won't work if you put card1 between card2 and card3.